mode/type/cluster/core/pom.xml

```diff
@@ -48,6 +48,11 @@
             <artifactId>shardingsphere-single-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
```

Comment:
`logback` dependency should not be added to the cluster module.

Reply:
I have resolved it, plz review it again